substrate/client/network/src/transport.rs

```diff
@@ -29,28 +29,21 @@ use libp2p::{
 };
 use std::{sync::Arc, time::Duration};
 
+// TODO: Create a wrapper similar to upstream `BandwidthTransport` that tracks sent/received bytes
```

Comment:
Is this blocked by `prometheus-client` crate update? Would be good to have an issue in substrate to not forget about this, thanks

Reply:
Not really, this tracker is ultimately used to display in/out bandwidth in the informer, which will be needed regardless of prometheus library used. Though we will be able to avoid this for metrics purposes with prometheus library upgrade indeed.